docs/network/interfaces_and_networks.md

```diff
@@ -91,61 +98,58 @@ definition.
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: bridge-test
+  name: linux-bridge-net-ipam
 spec:
   config: '{
       "cniVersion": "0.3.1",
-      "name": "bridge-test",
-      "type": "bridge",
-      "bridge": "br1",
-      "disableContainerInterface": true
+      "name": "mynet",
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "br10",
+          "disableContainerInterface": true,
+          "ipam": {
+            "type": "whereabouts",
```

Comment:
Isn't `whereabouts` another dependency one needs to install?

Reply:
yes it is, but @ormergi requested that I avoid using `host-local`  [here]( https://github.com/kubevirt/user-guide/pull/844#discussion_r1819184140)

Reply:
Adding some IPAM plugin from wth wild and thinking about IPAM on secondary networks is a distractions.

Just drop the IPAM usage, at least from the main multus description.
If IPAM is important on secondary network, then you can add it as its own section.

The example of a NAD with IPAM is given in `Multus as primary network provider` section.

Reply:
I'll go back to `host-local` it'll be consistent with the other example that you've mentioned and it works. I want to user to be able to copy paste from here and have a VM that can use its secondary interface